src/components/Comment/Content/index.tsx

```diff
@@ -79,7 +79,7 @@ const Content = styled.div<{ useLineClamp: boolean; maxLine: number }>`
   font-size: 14px;
   line-height: 20px;
   margin-top: 6px;
-  color: ${Colors.gray800};
+  color: ${gray800};
```

Comment:
font-size: 14px부터 body2로 쓸 수 있는 스타일입니당.
color는 기본 컬러가 gray800이라서 필요가 없습니다. 
```suggestion
  ${body2};
  margin-top: 6px
```

Reply:
흠 기존에 왜 이렇게 되어있던거지

Reply:
@tbvjaos510 참고해주세여

Reply:
넵!!!